deno.json

```diff
@@ -1,13 +1,18 @@
 {
   "lock": true,
   "tasks": {
+    "run-with-perms": "deno run --allow-env --allow-net --allow-sys=networkInterfaces,hostname --allow-read=. --allow-write=data-files",
```

Comment:
I'd rather have this be called `execute-with-permissions`, and I'd have to pull this down and further test all apps because I don't think this is all the permissions needed.